src/View/Sidebar.vala

```diff
@@ -441,6 +441,46 @@ namespace Marlin.Places {
             return iter;
         }
 
+        public override Gtk.TreeIter add_or_update_plugin_item (PluginItem item, Gtk.TreeIter? iter = null) {
+            Gtk.TreeIter parent;
+            store.get_iter (out parent, network_category_reference.get_path ());
```

Comment:
Restricts future plugin items to be in the network category?

Reply:
Sure, so I'll split add and update functions then so the params don't get too long.